share/completions/gojq.fish

```diff
@@ -0,0 +1,28 @@
+# Pure Go implementation of jq
+# https://github.com/itchyny/gojq
+
+complete -c gojq -s c -l compact-output -d "compact output, no pretty-print"
+complete -c gojq -s r -l raw-output -d "output raw strings without quotes"
+complete -c gojq -s j -l join-output -d "stop printing a newline after each output"
+complete -c gojq -s 0 -l nul-output -d "print NUL after each output"
+complete -c gojq -s C -l color-output -d "colorize output even if piped"
+complete -c gojq -s M -l monochrome-output -d "stop colorizing output"
+complete -c gojq -l yaml-output -d "output by YAML"
```

Comment:
"output *as* YAML"? The "by" sounds wrong to me

Reply:
Yes, I also stumbled upon this but wasn't sure because I just copied this over from `--help` output. I think you are right. I fixed it.